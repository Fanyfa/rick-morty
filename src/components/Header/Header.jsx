import React, { Component } from 'react';
import RickAndMorty from './img/Rick_and_Morty.png';
import styles from './styles.module.scss';

class Header extends Component {
  render() {
    return (
      <div className={styles.container}>
        <img src={RickAndMorty} alt="Morty" className={styles.image} />
      </div>
    );
  }
}

export default Header;
