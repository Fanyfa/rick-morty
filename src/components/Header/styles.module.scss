.container {
  text-align: center;

  .image {
    background-position: top center;
    background-size: contain;
    background-repeat: no-repeat;
    width: 100%;
    max-width: 539px;
  }
}
