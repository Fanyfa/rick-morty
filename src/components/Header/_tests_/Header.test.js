import Header from 'components/Header/';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<Header>', () => {
  it('renders', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
