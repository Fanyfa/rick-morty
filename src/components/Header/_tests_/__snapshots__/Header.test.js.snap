// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders 1`] = `
<div
  className="container"
>
  <img
    alt="Morty"
    className="image"
    src="Rick_and_Morty.png"
  />
</div>
`;
